imod/typing/grid.py

```diff
@@ -431,3 +433,20 @@ def is_transient_data_grid(
         if len(grid["time"]) > 1:
             return True
     return False
+
+
+@typedispatch
+def enforce_uda(grid: xr.DataArray) -> xu.UgridDataArray:
```

Comment:
rename to as_xugrid_dataarray?

Reply:
Good suggestion, was struggling a bit with naming here!